### PR TITLE
Bump TS and include parsed classname in type definition

### PR DESCRIPTION
**Pull request by JSimoni42**
Parse out classname to provide just the last portion to our templates.

Wanted to use option chaining for this, had to bump some other dependencies, as a result

### CODE REVIEW

**Comment by JSimoni42** [comment]
@ProdigySim do you know off the top of your head if tsbuildinfo system-specific?

**Comment by ProdigySim** [comment]
I think it's kind of a cache index... so there's probably a cache on the local system that it is supposed to match up with. We probably shouldn't check it in unless we see that is recommended. Add it to `.gitignore`?

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "@types/js-beautify": "^1.8.0",
     "@types/lodash": "^4.14.119",
     "@types/mustache": "^0.8.32",
-    "@types/node": "^10.12.18",
+    "@types/node": "^12.12.6",
```
Comment (ProdigySim) [inline package.json]: Why bump this to 12? Do you need to use 12 here? I think we are still on 10 for our official local builds...

**Review by ProdigySim — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (JSimoni42) [reply package.json]: I was running into this issue and it sounded like the solution was just to bump `@types/node`, https://github.com/microsoft/TypeScript/issues/32333 🤷. I bumped a few other packages over the course of this PR though, so lemme see if it's still an issue

Comment (JSimoni42) [reply package.json]: Yep, no longer necessary. Will bump this back down

**Pull request merged**